### PR TITLE
feat(Alert): Add the inline alert modifier

### PR DESCRIPTION
**Pull request by tlabaj**
#1934

**What**:

**Additional issues**:

### CODE REVIEW

**Comment by srambach** [comment]
It looks like the size of the icon is still 24px rather than reduced to 18px for the inline alert.

File: packages/patternfly-4/react-core/src/components/Alert/Alert.md [hunk 1]
```diff
@@ -175,3 +175,70 @@ class DangerAlert extends React.Component {
   }
 }
 ```
+
+## Inline alert types
+```js
+import React from 'react';
+import { Alert } from '@patternfly/react-core';
+
+class DangerAlert extends React.Component {
```
Comment (jschuler) [inline packages/patternfly-4/react-core/src/components/Alert/Alert.md]: perhaps change the class name here and also for the next new example

File: packages/patternfly-4/react-core/src/components/Alert/Alert.tsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ export interface AlertProps
   extends Omit<React.HTMLProps<HTMLDivElement>, 'action' | 'title'> {
   /** Adds Alert variant styles  */
   variant: 'success' | 'danger' | 'warning' | 'info';
+  isInline?: boolean;
```
Comment (jschuler) [inline packages/patternfly-4/react-core/src/components/Alert/Alert.tsx]: Missing comment description

**Comment by jschuler** [comment]
The icon sizes should be smaller for the inline alerts

File: yarn.lock [hunk 3]
```diff
@@ -1718,25 +1718,6 @@
   version "2.12.5"
   resolved "https://registry.yarnpkg.com/@patternfly/patternfly/-/patternfly-2.12.5.tgz#f2c5924deb875a8cde95e09e6d5c9022f00429da"
 
-"@patternfly/patternfly@2.8.2":
```
Comment (jschuler) [inline yarn.lock]: wouldn't expect yarn lock file to be in here

**Comment by mcarrano** [comment]
Looks good @tlabaj

**Pull request merged**